src/main/java/org/seqdoop/hadoop_bam/util/BGZFSplitGuesser.java

```diff
@@ -69,15 +77,24 @@ public long guessNextBGZFBlockStart(long beg, long end)
 		// the previous one, we need 0xfffe bytes for the start, and then 0xffff
 		// for the block we're looking for.
 
-		byte[] arr = new byte[2*0xffff - 1];
+		byte[] arr = new byte[0xffff];
```

Comment:
curious how this change interacts w the comment above? at the very least the latter seems outdated now?